### PR TITLE
*: nightly-build script

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/5331.

Docker images are serving at https://quay.io/repository/coreos/etcd-nightly.

Once can do:

```
sudo docker run quay.io/coreos/etcd-nightly:v2016051201 /usr/local/bin/etcd --version
sudo docker run quay.io/coreos/etcd-nightly:v2016051201 /bin/sh -c "/usr/local/bin/etcd --version"
```

### CODE REVIEW

**Comment by gyuho** [comment]
/cc @hongchaodeng @xiang90 @heyitsanthony 

This is mostly for testing purposes. Won't create any tags in main etcd repo.

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,15 @@
-FROM golang:onbuild
+FROM alpine:latest
```
Comment (xiang90) [inline Dockerfile]: why do we use alpine instead of golang:onbuild?

File: scripts/nightly.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```
Comment (xiang90) [inline scripts/nightly.sh]: who will run this script nightly?

File: Dockerfile (hunk 1 above)
Comment (gyuho) [inline Dockerfile]: @hongchaodeng needs linux based OS for Kubernetes test. `golang:onbuild` just copies the binary to the container image.

File: scripts/nightly.sh (hunk 2 above)
Comment (gyuho) [inline scripts/nightly.sh]: I can run or whenever @hongchaodeng needs a new image.

Quay has github trigger, if we need more automation.

**Comment by xiang90** [comment]
We should use https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd to build image for k8s. We should not put this inside etcd repo. k8s has its own assumptions on how to build the image.

**Comment by gyuho** [comment]
@xiang90 I wasn't aware of that.

@hongchaodeng it seems like Kubernetes needs to change their Dockerfile or change the testing manifest file to conform with the official k8s etcd images?

**Pull request closed without merging**